pkg/tsdb/influxdb/influxql/buffered/response_parser.go

```diff
@@ -50,8 +47,9 @@ func parse(buf io.Reader, statusCode int, query *models.Query) *backend.DataResp
 func parseJSON(buf io.Reader) (models.Response, error) {
 	var response models.Response
 
+	json := jsoniter.ConfigCompatibleWithStandardLibrary
+
 	dec := json.NewDecoder(buf)
-	dec.UseNumber()
```

Comment:
this was for reading number values as `json.Number` type. Then we have to convert them to `float64`. Here we skip that step. We decode them as `float64` directly.